doc/zero-to-binderhub/setup-binderhub.rst

```diff
@@ -183,6 +200,24 @@ where:
   If this is not provided, you may find BinderHub rebuilds images every launch instead of pulling them from the ACR.
   Suggestions for `<project-name>` could be `ACR_NAME` or the name you give your BinderHub.
 
+If you are using OVH Container Registry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want your BinderHub to push and pull images from an OVH Container Registry, then the `config.yaml` file will look the following::
+
+    config:
+      BinderHub:
+        use_registry: true
+        image_prefix: <hash>.<region>.container-registry.ovh.net/<prefix>-
```

Comment:
Mention that the prefix part should start with a project id within the Harbor registry?